code/src/main.rs

```diff
@@ -1,10 +1,28 @@
+use std::env::VarError;
+use std::process::exit;
 use actix_web::{get, post, Responder, HttpServer, HttpResponse, App, web};
 use tokio_postgres::{NoTls, Error};
 use tokio;
 
+fn check_env_key(key: &str) -> String {
+    let result:Result<String,VarError> = std::env::var(key);
+
+    if result.is_err() {
+        println!("Key {} is missing", key.to_string());
+
+        exit(1);
+    }
+
+    result.unwrap()
```

Comment:
Instead of checking with `if result.is_err()` you can use `result.expect("custom error message")` to print a custom error message. The `if` statement is redundant aswell then.